components/zendesk/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@pipedream/zendesk",
-  "version": "0.5.0",
+  "version": "0.5.1",
```

```suggestion
  "version": "0.6.0",
```
New `minor` version for a new component being added